clusterloader/vendor/github.com/ugorji/go/codec/test.py

```diff
@@ -98,15 +99,15 @@ def myStopRpcServer():
 def doRpcClientToPythonSvc(port):
     address = msgpackrpc.Address('localhost', port)
     client = msgpackrpc.Client(address, unpack_encoding='utf-8')
-    print client.call("Echo123", "A1", "B2", "C3")
-    print client.call("EchoStruct", {"A" :"Aa", "B":"Bb", "C":"Cc"})
+    print(client.call("Echo123", "A1", "B2", "C3"))
```

Comment:
You can't change whatever is in vendor/ directories. Please remove changes from this and the next file.

Reply:
I will change but...

This is the problem with vendoring in...  When you do that, you _own_ the code... Syntax Errors and all.  If you do not want to own the code, then do not vendor it in but pip install it instead.

Reply:
Also note that these changes are already landed in upstream.

Reply:
Also note that this whole directory was removed completely in the meantime :)

Reply:
Good news.  Thanks.